static/app/views/issueList/groupSearchViewTabs/editableTabTitle.tsx

```diff
@@ -51,40 +57,44 @@ function EditableTabTitle({
 
   useEffect(() => {
     if (isEditing) {
-      setTimeout(() => {
-        inputRef?.current?.focus();
-      }, 0);
+      requestAnimationFrame(() => {
+        inputRef.current?.focus();
+      });
     }
   }, [isEditing, inputRef]);
 
   const handleOnChange = (e: React.ChangeEvent<HTMLInputElement>) => {
     setInputValue(e.target.value);
   };
 
-  return isEditing ? (
-    <StyledGrowingInput
-      value={inputValue}
-      onChange={handleOnChange}
-      onKeyDown={handleOnKeyDown}
-      onBlur={handleOnBlur}
-      ref={inputRef}
-    />
-  ) : (
-    <div style={{height: '20px'}}> {label}</div>
+  return (
+    <FocusScope>
+      <StyledGrowingInput
```

Comment:
Seems a little wasteful to always render a GrowingInput - maybe we could render just the div if it's not selected?

Reply:
This created a little visual jitter when hitting rename because the div and growingInput didn't render the text in precisely the same position. 

https://github.com/user-attachments/assets/2978dac6-56d0-4dfe-b94f-b85997c2cf38
(This is in the issue stream now) 

Definitely a very small improvement, happy to add the div back if you think the minor change isn't worth rendering the GrowingInput

Reply:
I think it's good to render the input when the tab is selected, but I don't think we need to render the input when the tab isn't even selected (and thus cannot be edited), right?

The difference in styling may be due to the line-height

Reply:
That's true, okay I'll render just the div when its unselected in that case